source/documentation/unsubscribe-ern.html.md.erb

```diff
@@ -11,6 +11,7 @@ Use the POST endpoint [Unsubscribe an ERN to no longer receive Push/Pull notific
 
 ## Requirements
 
+The provided ERN must be present in the bearer token that you use for API authentication. If it is not, a 403 Forbidden error will be thrown.
```

Comment:
This isn't a great way to describe that the ern must be an enrolment. Any chance you could rephrase it to better reflect how it'd look from their perspective?